stdlib/public/runtime/MetadataLookup.cpp

```diff
@@ -3009,8 +2990,21 @@ swift_func_getParameterTypeInfo(
 
   SubstGenericParametersFromMetadata substFn(genericEnv, genericArguments);
 
-  DecodedMetadataBuilder builder(
```

Comment:
no other distributed impl funcs use this, so I'm hoping we're good. 

The only other thing `swift_func_getParameterCount` does not seem like it has any of these problems.